compose/src/main/kotlin/io/github/kakaocup/compose/node/core/BaseNode.kt

```diff
@@ -61,4 +62,20 @@ abstract class BaseNode<out T : BaseNode<T>> constructor(
             this,
         )
     }
+
+    /***
+     * Combines semantic matchers from all ancestor nodes
+     */
+    private fun combineSemanticMatchers(): SemanticsMatcher {
+        val semanticsMatcherList = mutableListOf<SemanticsMatcher>()
+        var currentParent = this.parentNode
```

Comment:
Is it possible to have non-current parent? Just past or next parent. In general can be just a `parent`